tests/unit/amo/components/TestSearchPage.js

```diff
@@ -48,44 +48,49 @@ describe('<SearchPage />', () => {
 
   it('renders a Paginate', () => {
     const root = render();
-    const paginator = findByTag(root, Paginate);
-    expect(paginator.props.count).toEqual(80);
-    expect(paginator.props.currentPage).toEqual(3);
-    expect(paginator.props.pathname).toEqual('/search/');
-    expect(paginator.props.queryParams).toEqual({ q: 'foo' });
+    const paginator = root.find(Paginate);
+    expect(paginator.prop('count')).toEqual(80);
```

Comment:
Oh, I didn't know that. I didn't like that the two things *seem* like they're the same thing and didn't like using both in the same test. I guess `toBe` is faster?